### PR TITLE
Migrate to JAX-RS from raw servlet

### PR DESCRIPTION
**Pull request by ebyhr**

### CODE REVIEW

**Comment by ebyhr** [comment]
@wyukawa Could you enable Travis CI integration in this repository when you have time?

File: build.gradle [hunk 1]
```diff
@@ -114,6 +114,7 @@ dependencies {
     compile 'com.google.inject.extensions:guice-servlet:4.0'
     compile 'org.codehaus.jackson:jackson-core-asl:1.9.13'
     compile 'org.codehaus.jackson:jackson-mapper-asl:1.9.13'
+    compile 'org.codehaus.jackson:jackson-jaxrs:1.9.13'
```
Comment (wyukawa) [inline build.gradle]: I'm not familiar with JSON library but is jackson-jaxrs necessary?

File: build.gradle [hunk 2]
```diff
@@ -129,6 +130,7 @@ dependencies {
     compile 'com.github.wyukawa.elasticsearch.unofficial.jdbc.driver:elasticsearch-jdbc-driver:0.0.9'
     compile 'org.jsoup:jsoup:1.11.3'
     compile 'mysql:mysql-connector-java:5.1.47'
+    compile 'com.owlike:genson:1.6'
```
Comment (wyukawa) [inline build.gradle]: I'm not familiar with JSON library but is genson necessary?

**Comment by wyukawa** [comment]
enabled Travis CI integration in this repository like https://github.com/yanagishima/yanagishima/pull/106/checks?check_run_id=253262182

File: build.gradle (hunk 1 above)
Comment (ebyhr) [reply build.gradle]: Sorry, it's my mistake during trial. Let me remove this library.

File: build.gradle (hunk 2 above)
Comment (ebyhr) [reply build.gradle]: This library is needed when we set json key name explicitly.
eg `@JsonProperty("formattedQuery")`

**Comment by wyukawa** [comment]
@ebyhr 
I merged and deployed but it didn't work.
http://localhost:8080 is 404

It works after I commented out the followng line
```
servletContextHandler.addServlet(new ServletHolder(new ServletContainer(new YanagishimaResourceConfig(config))), "/*");
```

So I reverted in https://github.com/yanagishima/yanagishima/commit/5e2f0c28b751166d1d49859a8624e10b68bda3b8

Could you please check?

**Comment by ebyhr** [comment]
Found the root cause. The default servlet path was overwritten with the above code. Once we swap the order as below, the endpoint will return the expected result. 
```java
        servletContextHandler.addServlet(new ServletHolder(new ServletContainer(new YanagishimaResourceConfig(config))), "/*");
        servletContextHandler.addServlet(DefaultServlet.class, "/");
```

**Pull request merged**